runtime/src/next/mod.rs

```diff
@@ -288,8 +255,8 @@ impl Router {
         tokio::task::spawn_blocking(move || {
             let mut iter = logs_stream.bytes().filter_map(Result::ok);
 
-            while let Some(log) = Log::from_bytes(&mut iter) {
-                logs_tx.blocking_send(Ok(log.into())).expect("to send log");
+            while let Some(_log) = Log::from_bytes(&mut iter) {
+                // TODO: send to the log server here
```

Comment:
Are you thinking we should do this in a follow-up?

Reply:
Yip, the wasm needs some other fixes first :sweat_smile: